README.md

```diff
@@ -23,7 +23,7 @@
   <a href="#-how-to-contribute">How to Contribute</a>
   <span> · </span>
   <a href="#current-contributors">Current Contributors</a>
```

Comment:
These will need to be updated to link to the new file. 👍

Reply:
Done and just updated the PR with the new links